### PR TITLE
Golf clubs can be picked up from the racks

### PR DESCRIPTION
**Pull request by chev2**
People have been asking this for a while. Instead of having the golf club auto-equip, make it so that a player has to actually walk up to the rack to grab one. I added some more golf racks onto the map (7 in total now) so that there are some in the caverns as well.

If you can please remove whatever lua code causes the auto-equip, please do.

I also want to make this as newbie-friendly as possible. A chat message when walking into one of the golf areas or a ply:Notify saying to grab a golf club from one of the racks should work.

### CODE REVIEW

File: lua/entities/golfrack.lua [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	end
+end
+
+function ENT:Draw()
+    self:DrawModel()
+end
+
+function ENT:Think()
+    if SERVER then
+        for k, v in pairs(ents.FindInSphere(self:GetPos(), 60)) do
```
Comment (swampservers) [inline lua/entities/golfrack.lua]: This is way too expensive. Can you do it with entity:Touch or :Use instead?

**Comment by swampservers** [comment]
Great change thanks!

File: lua/entities/golfrack.lua (hunk 1 above)
Comment (chev2) [reply lua/entities/golfrack.lua]: Sure. That's why I put in a NextThink(), I figured it would be performance-heavy

**Comment by swampservers** [comment]
thanks. can you test this out before i remove the code to automatically give you it? you can just /drop the club then try the stands. In particular please test the no-drawed stands that are on the surface to make sure you can still touch them.

**Pull request merged**